bin/MadGraph5_aMCatNLO/cards/production/13TeV/TTZJets/TTZJets_5f_LO_MLM/ttZ01j_5f_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+import model sm-ckm_no_b_mass
+
+define p = p b b~
+define j = j b b~
+
+generate p p > t t~ z, z > j j @0
+add process p p > t t~ z, z > vl vl~ @1
+add process p p > t t~ l+ l- @2
+add process p p > t t~ vl vl~ @3
```

Comment:
vl vl~ decays are double counted here.  (because you have it defined both with and without the decay chain syntax)

Suggest keeping only the t t~ z, z > vl vl~ version.